src/opera/pge/dist_s1/dist_s1_pge.py

```diff
@@ -85,33 +86,33 @@ class DistS1PostProcessorMixin(PostProcessorMixin):
 
     _valid_layer_names = _main_output_layer_names + _confirmation_db_output_layer_names
 
-    def _validate_outputs(self):
-        product_id_pattern = (r'(?P<id>(?P<project>OPERA)_(?P<level>L3)_(?P<product_type>DIST(-ALERT)?)-(?P<source>S1)_'
-                              r'(?P<tile_id>T[^\W_]{5})_(?P<acquisition_ts>(?P<acq_year>\d{4})(?P<acq_month>\d{2})'
-                              r'(?P<acq_day>\d{2})T(?P<acq_hour>\d{2})(?P<acq_minute>\d{2})(?P<acq_second>\d{2})Z)_'
-                              r'(?P<creation_ts>(?P<cre_year>\d{4})(?P<cre_month>\d{2})(?P<cre_day>\d{2})T'
-                              r'(?P<cre_hour>\d{2})(?P<cre_minute>\d{2})(?P<cre_second>\d{2})Z)_(?P<sensor>S1[AC]?)_'
-                              r'(?P<spacing>30)_(?P<product_version>v\d+[.]\d+))')
+    _product_id_pattern = (r'(?P<id>(?P<project>OPERA)_(?P<level>L3)_(?P<product_type>DIST(-ALERT)?)-(?P<source>S1)_'
```

Comment:
I just realized that we can simplify this regex a bit, since I don't think we'll have a need for granular access to the parsed timestamps, meaning we could remove the `?<acq_hour/minute/second>` and `?P<cre_hour/minute/second>` and just keep the full capture for `?p<acquisition_ts>` and `?p<creation_ts>`

Reply:
Trimmed it down quite nicely, thanks